src/roles/common/tasks/vsd-generate-transfer-certificates.yml

```diff
@@ -40,8 +40,13 @@
       CERT_PASSWORD: "{{ certificate_password }}"
     when: "'4.0.4' not in vsd_version and scp_user is not defined and scp_location is not defined"
 
+  - name: Check vsc hostname existence in IPv6
```

Comment:
What is the requirement for hostname under IPv6? In your vsc.j2 file, you set the hostname to IP address if hostname isn't defined in the vscs.yml file. Is it okay if hostname is an IP address?

Reply:
There is a bug for the vsc, in the ipv6 mode, the hostname is required, I am not sure if they have resolved this bug. I will check it

Reply:
Resolved it